initializr-generator/src/test/java/io/spring/initializr/generator/buildsystem/maven/MavenDistributionManagementTests.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.initializr.generator.buildsystem.maven;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for {@link MavenDistributionManagement}
+ *
+ * @author Joachim Pasquali
+ */
+public class MavenDistributionManagementTests {
+
+	private static final String TEST_URL = "testURL";
+
+	@Test
+	void emptyTest() {
```

Comment:
I am not super happy with the consistency of the test names in the current codebase (largely my faut I should confess), but the "Test" prefix is not something we do. Can you look at other similar "container" test for inspiration please?